adjust glue for #1907 (#1945)
latex3/latex2e: base/doc/ltnews43.tex

%\section{Changes to packages in the \pkg{graphics} category}

\section{Changes to packages in the \pkg{tools} category}
\subsection{Adjustment to the glue used by \pkg{longtable}}

> Recent LaTeX releases produce ignored warnings about infinite glue shrinkage,

Reviewer: ```suggestion
Recent \LaTeX{} releases produce ignored warnings about infinite glue shrinkage,
```

> The glue in `pkg{longtable} has been adjusted to on;y use a finite shrink component.

Reviewer: ```suggestion
The glue in `pkg{longtable} has been adjusted to only use a finite shrink component.
```
